packages/123done/static/js/123done.js

```diff
@@ -95,7 +95,7 @@ $(document).ready(function () {
   $.getJSON(
     `${paymentConfig.contentEnv}metrics-flow?form_type=button&utm_campaign=123done`
   ).done(function (data) {
-    $('.btn-subscribe-pwdless').each(function (index) {
+    $('.btn-subscribe').each(function (index) {
```

Comment:
This is only relevant change. The html above is updated for this, guess prettier hasn't touched the file.